README.md

```diff
@@ -1,45 +1,51 @@
-- [Tether](http://tether.io/)  (tested with 1.3.7)
-- [React](https://facebook.github.io/react/) (tested with 15.3.1)
-- [React DOM](https://facebook.github.io/react/) (tested with 15.3.1)
-- [Underscore](http://underscorejs.org/)  (tested with 1.8.3)
-
-### Building
-Make sure you have Node JS and NPM installed.
-Run `npm install` to get the required dependencies and build the librairy.
-
-### Running documentation locally
-To build and run the doc locally, simply run `npm start`.
-
-### Running the tests suite
-Run `npm test` to run all tests and get the code coverage!
-
-## Contributing
-1. Search the issues, if it is not already there, add one.
-2. Fork the repository
-3. Code Code Code
-4. Submit a pull request
-5. Wait for some nice guy to review and merge
-
-## License
-Vapor is distributed under [MIT license](LICENSE).
-
+# React-Vapor
+
+[![Build Status](https://travis-ci.org/coveo/react-vapor.svg?branch=master)](https://travis-ci.org/coveo/react-vapor)
+[![codecov](https://codecov.io/gh/coveo/react-vapor/branch/master/graph/badge.svg)](https://codecov.io/gh/coveo/react-vapor)
+[![bitHound Overall Score](https://www.bithound.io/github/coveo/react-vapor/badges/score.svg)](https://www.bithound.io/github/coveo/react-vapor)
+[![bitHound Dependencies](https://www.bithound.io/github/coveo/react-vapor/badges/dependencies.svg)](https://www.bithound.io/github/coveo/react-vapor/master/dependencies/npm)
+[![bitHound Dev Dependencies](https://www.bithound.io/github/coveo/react-vapor/badges/devDependencies.svg)](https://www.bithound.io/github/coveo/react-vapor/master/dependencies/npm)
+[![Greenkeeper badge](https://badges.greenkeeper.io/coveo/react-vapor.svg)](https://greenkeeper.io/)
+[![Npm total downloads badge](https://img.shields.io/npm/dt/react-vapor.svg)](https://www.npmjs.com/package/react-vapor)
+
+Vapor CSS components implemented with React!
+
+## Install
+```sh
+npm install react-vapor
+```
+> The project is at its early stages, some components can still have _lots_ of breaking changes between versions.
+
+### Dependencies
+Those are required in order to use react-vapor in your own project:
+- [Tether](http://tether.io/)  (tested with 1.3.7)
+- [React](https://facebook.github.io/react/) (tested with 15.3.1)
+- [React DOM](https://facebook.github.io/react/) (tested with 15.3.1)
+- [Underscore](http://underscorejs.org/)  (tested with 1.8.3)
+
+### Building
+Make sure you have Node JS and NPM installed.
+Run `npm install` to get the required dependencies and build the librairy.
+
+### Running documentation locally
+To build and run the doc locally, simply run `npm start`.
+
+### Running the tests suite
+Run `npm test` to run all tests and get the code coverage!
+
+## Contributing
+1. Search the issues, if it is not already there, add one.
+2. Fork the repository
+3. Code Code Code
+4. Submit a pull request
+5. Wait for some nice guy to review and merge
+
+** Folder stucture
+1. create a folder in sentense case for your new component

```

Comment:
Should be _camel case_ instead of _sentence case_
* Sentence case: _Submit a pull request_
* Title case: _Submit A Pull Request_
* Camel case: _SubmitAPullRequest_